conf/bafu.conf.part

```diff
@@ -547,18 +547,180 @@ source src_ch_bafu_bundesinventare-auen : def_searchable_features
 {
     sql_db = bafu_dev
     sql_query = \
-        SELECT gid::bigint as id \
-            , au_name as label \
+        SELECT bgdi_id::bigint as id \
+            , name as label \
             , 'feature' as origin \
-            , au_obj::text||' '||remove_accents(au_name) as detail \
+            , objnummer::text||' '||remove_accents(coalesce(name,' ')||' '||coalesce(auen_type_de,' ')||' '||coalesce(auen_type_fr,' ')||' '||coalesce(auen_type_it,' ')) as detail \
             , 'ch.bafu.bundesinventare-auen' as layer \
             , quadindex(the_geom) as geom_quadindex \
             , st_y(st_transform(st_ClosestPoint(the_geom,st_centroid(the_geom)),4326)) as lat \
             , st_x(st_transform(st_ClosestPoint(the_geom,st_centroid(the_geom)),4326)) as lon \
             , box2d(st_transform(the_geom, 21781)) as geom_st_box2d \
             , box2d(st_transform(the_geom, 2056)) as geom_st_box2d_lv95 \
-            , gid::text as feature_id \
-        from bundinv.au
+            , bgdi_id::text as feature_id \
+        from bundinv.auen
+}
+
+source src_ch_bafu_bundesinventare-auen_anhang2 : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+            , name as label \
+            , 'feature' as origin \
+            , objnummer::text||' '||remove_accents(coalesce(name,' ')||' '||coalesce(typ_de,' ')||' '||coalesce(typ_fr,' ')||' '||coalesce(typ_it,' ')) as detail \
```

Comment:
concat_ws is safefer and avoids multiple spaces when an attribute's value is null.
`remove_accent(concat_ws(' ', objnummer::text, name, typ_de, typ_fr, typ_it))`